validation_test.go

```diff
@@ -235,6 +235,7 @@ func TestValidation_Param_Value(t *testing.T) {
 			params: map[string]Parameter{
 				"param1": {Validations: []Validation{
 					ValidationRequired{},
+					ValidationRequired{},
```

Comment:
required validation is duplicated here

Reply:
Yeah, because that param is _super important_ 😂